### PR TITLE
Fix group node copy paste

### PR DESCRIPTION
**Pull request by christian-byrne**
Two issues with group node copy/paste:

1. Group nodes unregister themselves when `app.clean` is called (pressing "Clear Workflow", loading a blank workflow), preventing copy-pasting between workflows afterwards.
    **Replicate**: create group node, "Workflow" -> "New", try to copy/paste group node between workflows.
   **Fix**: Remove unregistration. It no longer makes sense when multiple workflows can be open simultaneously.

3. Group nodes are added to `app.graph.extra` during construction but not after copy/pasting. 
    **Replicate**: create group node, paste to another workflow, reload or save/load that workflow.
   **Fix**: Add to `app.graph.extra` during node creation callback.

### CODE REVIEW

File: browser_tests/ComfyPage.ts [hunk 1]
```diff
@@ -1066,7 +1066,7 @@ export class NodeReference {
     let clickPos: Position
     switch (position) {
       case 'title':
-        clickPos = { x: nodePos.x + nodeSize.width / 2, y: nodePos.y + 15 }
+        clickPos = { x: nodePos.x + nodeSize.width / 2, y: nodePos.y + 1 }
```
Comment (huchenlei) [inline browser_tests/ComfyPage.ts]: Can you explain why this change is necessary? I think `+1` is clicking the node body not title. You can add a new position type `body` here if necessary.

**Review by huchenlei — CHANGES_REQUESTED** [review]

File: browser_tests/ComfyPage.ts (hunk 1 above)
Comment (christian-byrne) [reply browser_tests/ComfyPage.ts]: Yes, I was meaning to ask if 'title' meant the text or the title bar.

Using `+ 15` resulted in clicking the first-most widget of group nodes:

Selection_464

Comment (christian-byrne) [reply browser_tests/ComfyPage.ts]: I will create a body option

Comment (huchenlei) [reply browser_tests/ComfyPage.ts]: I think the original value was `-15`?

**Comment by huchenlei** [comment]
Can you do a rebase instead of merge? Currently it captures many changes not belonging to this PR.

File: browser_tests/ComfyPage.ts (hunk 1 above)
Comment (christian-byrne) [reply browser_tests/ComfyPage.ts]: Seems to have been changed to `- 15` 3 hours ago in #1022

**Review by huchenlei — APPROVED** [review]

**Comment by christian-byrne** [comment]
> Can you do a rebase instead of merge? Currently it captures many changes not belonging to this PR.

Sorry - Thanks for doing it.
